src/main/resources/assets/voidfog/lang/ru_ru.json

```diff
@@ -2,11 +2,13 @@
 	"menu.voidfog.title": "Параметры тумана пустоты",
 	"menu.voidfog.enabled": "Включить туман",
 	"menu.voidfog.creative": "Отключить в креативе",
+	"menu.voidfog.scale": "Scale With Difficulty",
 	"menu.voidfog.torches": "Спрятаться в освещенных местах",
 	"menu.voidfog.particles.min": "Без частиц",
 	"menu.voidfog.particles.max": "ВСЕ частицы",
 	"menu.voidfog.particles.default": "Частицы: По умолчанию",
 	"menu.voidfog.particles": "Частицы: %d",
+	"menu.voidfog.fogheight": "Fog Height: %d",
 	"menu.voidfog.bigboi": "Режим большого мальчика",
 	"menu.voidfog.bigboi.tooltip": "Делает игру еще более пугающей.\r\nВы большой парень?"
 }
```

Comment:
Please remove the english entries from this file. It should only contain the russian translations of the ones in en_us!

Reply:
My bad - put that in there to test something and forgot to fix it.

Reply:
Fixed in [e0240d0](https://github.com/Sollace/Void-Fog/pull/13/commits/e0240d023896046a37fd7e0bb7a041be095248b1)